docs/usage/SYNC_OPTIONS.md

```diff
@@ -0,0 +1,62 @@
+# Sync Options
+
+#### `errorCallBack`
+a callback that is called whenever an error event occurs during the sync process.
+
+#### `warningCallBack` 
+a callback that is called whenever a warning event occurs during the sync process.
+
+#### `beforeUpdateCallback`
+a filter function which can be applied on a generated list of update actions. It allows the user to intercept 
```

Comment:
```suggestion
a filter function, which can be applied on a generated list of update actions. It allows a user to intercept 
```

Reply:
I don't this comma is grammatically correct: https://en.oxforddictionaries.com/punctuation/comma

Reply:
True

Reply:
True. My bad, sorry